### DISCUSSION TITLE
Clarification in documentation for otPlatAlarmMilliStartAt

### DISCUSSION THREAD

**Discussion by suveshpratapa** [openthread/openthread#9765] [General]
The [documentation](https://github.com/openthread/openthread/blob/main/include/openthread/platform/alarm-milli.h#L66) for `otPlatAlarmMilliStartAt` has:
```
 * Set the alarm to fire at @p aDt milliseconds after @p aT0.
 *
 * For @p aT0 the platform MUST support all values in [0, 2^32-1].
 * For @p aDt, the platform MUST support all values in [0, 2^31-1].
```

This implies that `aT0 + aDt` could be larger than UINT32_MAX.

However, a [previous discussion](https://github.com/openthread/openthread/pull/9007#issuecomment-1544322914) we had implied that all otPlatAlarmMilli APIs are 32-bit timer APIs. Should we fix the documentation, since it implies we might have to handle a 64-bit value, or is it implicit that the implementation handles wrapping?

**Comment by jwhui** [comment]
The implementation of `otPlatAlarmMilli` can satisfy the documented requirements using a 32-bit timer by properly handling the wrap.
